src/get-started/java-binding.md

```diff
@@ -1027,6 +1033,10 @@ services without stopping the nodes in the network. This can be done by sending
 particular transactions to the built-in Supervisor service. To simplify this
 process for the users, `exonum-launcher` is recommended to be used.
 
+Compiled services JAR files must be placed into the directory `artifacts-path`
```

Comment:
Would it be better to call it here 'the artifacts directory'?

"... into the artifacts directory ([configured with] `artifacts-path`)."